ARCs/arc-0060.md

```diff
@@ -0,0 +1,330 @@
+
+```tsx
+export type StdData = string;
+```
+
+The `StdData` must be validated with respect the JSON schema provided with `metadata` (defined below).
+
+#### Interface `Ed25519Pk`
+
+An `Ed25519Pk` object is a 32-byte public key, point of the ed25519 elliptic curve. The key **MUST NOT** be transformed into an Algorand address.
+
+```tsx
+export type Ed25519Pk = Uint8Array;
+```
+
+> The wallet **MAY** want to operate with standard Algorand address directly. The transformation from a generic key to an Algorand address is left to the implementation. See <a href="https://developer.algorand.org/docs/get-details/accounts/#transformation-public-key-to-algorand-address">Public Key to Algorand Address</a> section of the developer portal.
+
+#### Interface `SignedDataStr`
+
+`SignedDataStr` is the produced 64-byte array, ed25519 digital signature, of the signed data.
+
+```tsx
+export type SignedDataStr = Uint8Array;
+```
+
+#### Enum `ScopeType`
+
+The `ScopeType` enumerates constant strings with the scope of a signing action.
+
+This ARC introduces two scope types.
+
+| ScopeType | Description |
+| --- | --- |
+| MSGSIG | Signature of a simple message. This is the most generic scope. |
+| LSIG | Signature of an Algorand program for delegation. |
+| ... | ... |
+
+Any extension of this ARC **SHOULD** adopt the `ScopeType` above, or introduce a new scope.
+
+#### Interface `StdSignMetadata`
+
+A `StdSignMetadata` object specifies metadata of a `StdSignData` that is being signed.
+
+```tsx
+export interface StdSignMetadata {
+    /**
+    * The scope value of the signing data request.
+    */
+    scope: ScopeType;
+
+    /**
+    * Canonical representation of the JSON schema for the signing data.
+    */
+    schema: string;
+
+    /**
+    * Optional message explaining the reason for the signature.
+    */
+    message?: string;
+
+    /**
+    * Optional encoding used to represent the signing data.
+    */
+    encoding?: string;
+}
+```
+
+If the `encoding` is not specified, the `StdData` object should be UTF-8 encoded following the <a href="https://www.rfc-editor.org/rfc/rfc8785">RFC 8785</a>.
+
+##### Simple Message Data JSON Schema
+
+The JSON schema for simple message signing. The `StdData` **MUST** be an object of this schema.
+
+> The signing data JSON schema is inspired by the schema proposed with <a href="https://eips.ethereum.org/EIPS/eip-712">EIP-712: Typed structured data hashing and signing proposal</a>.
+
+```json
+{
+  "type": "object",
+  "properties": {
+    "ARC60Domain": {
+      "type": "string",
+      "description": "The ARC-60 domain separator"
+    },
+    "bytes": {
+      "type": "string",
+      "description": "Byte string being signed. Used with ScopeType==MSGSIG."
+    },
+  "required": ["ARC60Domain", "bytes"]
+}
```

Comment:
Is this outdated here? It seems in the file `simple-schema.json` you replaced the `bytes` (here type string) with an integer array as per @ehanoc's suggestion.